llvm/test/Analysis/LoopAccessAnalysis/different_strides.ll

```diff
@@ -0,0 +1,80 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 5
+; RUN: opt --disable-output -mtriple=x86_64 --passes="print<access-info>" %s 2>&1 | FileCheck %s
```

Comment:
is `mtriple` needed?

Reply:
Yes, otherwise LAA returns unknown dependency. I suppose it is because of the `MaxVFInBits < MaxTargetVectorWidthInBits` check at the end of `isDependent`, which is false because MaxTargetVectorWidthInBits has not updated with target info and remains unsigned max.